test/box-tap/several-listening-sockets.test.lua

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env tarantool
+local tap = require('tap')
+local net_box = require('net.box')
+local os = require('os')
+local fio = require('fio')
+
+-- Create table which contain several listening uri,
+-- according to template @a addr. It's simple adds
+-- capital letters in alphabetical order to the
+-- template.
+local function create_uri_table(addr, count)
+    local uris_table = {}
+    local path_table = {}
+    local ascii_A = string.byte('A')
+    for i = 1, count do
+        local ascii_code = ascii_A + i - 1
+        local letter = string.char(ascii_code)
+        path_table[i] = addr .. letter
+        uris_table[i] = "unix/:" .. addr .. letter
+    end
+    return uris_table, path_table
+end
+local function check_connection(port)
+    local conn = net_box.connect(port)
+    local rc = conn:ping()
+    conn:close()
+    return rc
+end
+
+local test = tap.test('gh-6535-listen-update-numeric-uri')
+test:plan(1 + 3 + 5 + (4 * 5 + 5) + 7 + 3)
```

Comment:
Maybe just put a single number here? This notation does not simplify its update really. Anyway update nearly always happens by running the test, looking at the number in the failure message, and putting it into the plan.